tutorhastexo/patches/openedx-common-settings

```diff
@@ -0,0 +1,2 @@
+XBLOCK_SETTINGS["hastexo"] = {{ HASTEXO_XBLOCK_SETTINGS|tojson(indent=4) }}
```

Comment:
Is the `tojson()` filter still appropriate here? This is a Python dictionary, no?

Reply:
Yes, I can remove that.